src/AltinnCore/Runtime/gulpfile.js

```diff
@@ -0,0 +1,78 @@
+const gulp = require('gulp');
+const run = require('gulp-run-command').default;
+const chokidar = require('chokidar');
+const fs = require('fs');
+
+const jsRuntimeFile = '../../react-apps/applications/runtime/dist/runtime.js';
+const cssRuntimeFile = '../../react-apps/applications/runtime/dist/runtime.css';
+
+let jsWatcher = null;
+let cssWatcher = null;
+
+function copyReactJs(cb) {
+  copyRuntimeJs();
+  cb();
+  return;
+}
+
+function copyReactCss(cb) {
+  copyRuntimeCss();
+  cb();
+  return;
+}
+
+function copyRuntimeJs() {
+  setTimeout(function () {
+    gulp.src(jsRuntimeFile).pipe(gulp.dest('../Designer/wwwroot/designer/js/react'));
+  }, 1000);
+  return;
+}
+
+function copyRuntimeCss() {
+  setTimeout(function () {
+    gulp.src(cssRuntimeFile).pipe(gulp.dest('../Designer/wwwroot/designer/css/react'));
+  }, 1000);
+  return;
+}
+
+function setupWatchers(cb) {
+  var checkRuntimeJsFile = setInterval(function () {
+    if (fs.existsSync(jsRuntimeFile)) {
+      jsWatcher = chokidar.watch(jsRuntimeFile);
+      // jsWatcher.on('ready', copyReactJs);
+      jsWatcher.on('change', copyRuntimeJs);
+      clearInterval(checkRuntimeJsFile);
+    }
+  }, 1000);
+
+  var checkRuntimeCssFile = setInterval(function () {
+    if (fs.existsSync(cssRuntimeFile)) {
+      cssWatcher = chokidar.watch(cssRuntimeFile);
+      // cssWatcher.on('ready', copyReactCss);
+      cssWatcher.on('change', copyRuntimeCss);
+      clearInterval(checkRuntimeCssFile);
+    }
+  }, 1000);
+
+  cb();
+}
+
+gulp.task('copy-files', gulp.series(
+  copyReactJs,
+  copyReactCss
+));
+
+gulp.task('develop', gulp.parallel(
+  setupWatchers,
+  run('dotnet run'),
+  run('npm run webpack-watch', {
+    cwd: '../../react-apps/applications/runtime',
+  })
+));
+
+gulp.task('default', gulp.series([
+  run('npm run build', {
+    cwd: '../../react-apps/applications/runtime',
+  }),
+  'copy-files'
+]));
```

Comment:
Is there really a need for a gulp-file in runtime? The `runtime.js` file that is being used comes from the service repo, not from the designer. So you still need to copy over the file to the service repo that is being used to have any effect. You would get the same result by running the webpack dev build directly from the react-apps/applications/runtime folder and copying the runtime.js file from dist.

Reply:
I guess, but then you would have to run two independent scripts. I just would find it easier to run npm run gulp-develop rather then running two separate commands when developing. But this is just a preference. I also like the consistency for both Designer and Runtime development.

Reply:
Fair enough :)
Can you update the flight rules after this PR goes through with description on where to copy runtime.js to/from when testing runtime locally?

Reply:
Will do!